### DISCUSSION TITLE
How to change logging levels

### DISCUSSION THREAD

**Discussion by ambroisie** [jonaswinkler/paperless-ng#1477] [Support]
I am trying to diagnose an issue, and would like to know if there is an environment variable I could add to my systemd service to change the log levels.

**Comment by torbengb** [comment]
In paperless.conf there is the setting `#PAPERLESS_DEBUG=false` that you could uncomment and set to true. What I only just now realized is [this line in the docs](https://paperless-ng.readthedocs.io/en/latest/configuration.html#configuration): "If you run paperless **on docker, paperless.conf is not used.** Rather, configure paperless by copying necessary options to docker-compose.env."

That would the be:
```
      PAPERLESS_DEBUG: false
```

**Reply by ambroisie** [reply]
I understood this as something that should exclusively be used for development purposes, as it disables important security features.

I am wondering if there is anything that gets logged when using this, that wouldn't during normal usage, an if so how to log it for debugging purposes while also-ran not taking unnecessary risk.
